### ISSUE TITLE
Download entries feature not working properly 

### ISSUE THREAD

**Issue by gopalakrishnaa** [hatnote/montage#235]
### Expected behaviour:
The event coordinator should be able to download the results in Image entries format with username and image file name as well as Download results in scores format with user name as well. 

### Current behaviour:
The event coordinator unable to download the results in Image entries format. While clicking both buttons are giving the same results. Event coordinator needs to go to results and find user names manually. 

### Details:
After the third round of voting completed by juries, the event coordinator can download results in two formats. One option is Download Entries option and another option is Download Results option. Expected behaviour is to get results in Excel entries format with scores and Winning images in the winning order. 

Anyone from Hatnote can solve this? Please help @mahmoud @yarl @slaporte

**Comment by mahmoud** [comment]
Hi @gopalakrishnaa thanks for the report. I just attempted to reproduce this on a finalized campaign and was able to get two different formats. In this screenshot, you can see the files have different names and sizes:

![image](https://github.com/user-attachments/assets/a10cc579-0cb1-4948-896e-61b92b5e6bb4)

Can you confirm that the file contents are the same, and if so, can you share a campaign/round ID/URL for the issue you're experiencing? We haven't had any recent changes or reports regarding this, and it's a very core functionality, so I expect it to work. Thanks!

**Comment by gopalakrishnaa** [comment]
I am one of the Jury members for **Tribal Culture Photography Competition 2024**. The coordinator reported that she cannot download in two different formats. As she was trying to download she was getting the same format. If you can give me full access I will try to reproduce it again.

**Comment by mahmoud** [comment]
I tried that specific campaign and round and was not able to reproduce the issue. I've added you as a coordinator for the campaign to confirm the same. Use the permission appropriately.

I also looked at the frontend and backend code, both are straightforward with no special handling for active vs finalized campaigns or other corner cases that might be at play. My guess is that the files might have gotten mixed up after download.
